roguelike-gruid-project/internal/ui/tiles_test.go

```
@@ -0,0 +1,139 @@
+	if defaultConfig.Enabled {
+		t.Error("Default tile config should have tiles disabled")
+	}
+	if defaultConfig.TileSize != 16 {
+		t.Errorf("Expected default tile size 16, got %d", defaultConfig.TileSize)
+	}
+	if defaultConfig.ScaleFactor != 1.0 {
+		t.Errorf("Expected default scale factor 1.0, got %f", defaultConfig.ScaleFactor)
+	}
+	
+	// Test configuration validation
+	if !config.ValidateTilesetPath("") {
```

golangci-lint (1.64.8)

SA9003: empty branch

(staticcheck)